Webapp/sources/blueprints/reaction_table/routes.py

```diff
@@ -18,17 +18,17 @@
 
 from . import reaction_table_bp
 
-if not current_app.config["DEBUG"]:
-    print("Trying to import STOUT")
-    try:
-        from STOUT import translate_forward
-
-        print("STOUT successfully imported")
-    except Exception:
-        print("Failed to import STOUT")
-        pass
-else:
-    print("Application in debug mode. Not importing STOUT")
+# if not current_app.config["DEBUG"]:
```

Comment:
one day you'll have a future PR for "commented code tidied" :)
but glad to see the back of stout